### PR TITLE
remove unused cloudprovider param from the ProbeControllerVolumePlugins func

### PR DESCRIPTION
**Pull request by carlory**
#### What type of PR is this?

/kind cleanup

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: cmd/kube-controller-manager/app/plugins.go [hunk 1]
```diff
@@ -25,8 +25,6 @@ import (
 
 	"k8s.io/klog/v2"
 
-	// Cloud providers
-	cloudprovider "k8s.io/cloud-provider"
 	// ensure the cloud providers are installed
 	_ "k8s.io/kubernetes/pkg/cloudprovider/providers"
```
Comment (aojea) [inline cmd/kube-controller-manager/app/plugins.go:29]: is this needed?

Comment (carlory) [reply cmd/kube-controller-manager/app/plugins.go:29]: Yes, it's needed. It  populates the private variable providers map in the `k8s.io/cloud-provider` and is used to create the cloud provider instance for the given `--cloud-provider` flag in the kube-controller-manager. 

So we need keep it until the [KEP-2395](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cloud-provider/2395-removing-in-tree-cloud-providers) is GAed.

**Pull request merged**